src/ParallelAlgorithms/Events/ObserveTimeStep.hpp

```diff
@@ -50,6 +51,34 @@ using ObserveTimeStep =
     ::Registration::Registrar<Events::ObserveTimeStep, Metavariables>;
 }  // namespace Registrars
 
+namespace {
+struct FormatTimeOutput
+    : tt::ConformsTo<observers::protocols::ReductionDataFormatter> {
+  using reduction_data = Parallel::ReductionData<
+      Parallel::ReductionDatum<double, funcl::AssertEqual<>>,
+      Parallel::ReductionDatum<size_t, funcl::Plus<>>,
+      Parallel::ReductionDatum<double, funcl::AssertEqual<>>,
+      Parallel::ReductionDatum<double, funcl::Min<>>,
+      Parallel::ReductionDatum<double, funcl::Max<>>,
+      Parallel::ReductionDatum<
+          double, funcl::Plus<>,
+          funcl::Divides<funcl::Literal<1, double>, funcl::Divides<>>,
+          std::index_sequence<1>>>;
+  std::string operator()(const double time, const size_t /* num_points */,
+                         const double /* slab_size */,
+                         const double /* min_time_step */,
+                         const double /* max_time_step */,
+                         const double /* effective_time_step */) const
+      noexcept {
+    return "Current time: " + std::to_string(time);
```

Comment:
Would it be useful to print some of the other data? Up to you and others who have an opinion on this :)